### PR TITLE
Add database cleanup

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cciutils/scripts/k8s/db.py [hunk 1]
```diff
@@ -14,9 +14,16 @@ def main() -> None:
     """Get some logs to from k8s."""
     parser = argparse.ArgumentParser(description="Get some logs to from k8s.")
     parser.add_argument("--script", help="The script used to initialise the database")
+    parser.add_argument("--cleanup", help="Drop the database")
 
     args = parser.parse_args()
 
+    if args.cleanup:
+        _print("::group::Cleanup the database")
+        subprocess.run(["helm", "uninstall", "test-pq"], check=False)
```
Comment (danduk82) [inline c2cciutils/scripts/k8s/db.py:23]: typo: `test-pq` -> `test-pg`

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -97,6 +97,10 @@ jobs:
         run: c2cciutils-k8s-logs
         if: always()
 
+      - name: Clean the database
+        run: helm uninstall test-pg || true
```
Comment (danduk82) [inline .github/workflows/main.yaml:101]: call the script here instead no?

Comment (sbrunner) [reply .github/workflows/main.yaml:101]: Oups :-)

File: c2cciutils/scripts/k8s/db.py (hunk 1 above)
Comment (sbrunner) [reply c2cciutils/scripts/k8s/db.py:23]: Oups :-)

**Pull request merged**